daemon/api.go

```diff
@@ -296,6 +296,7 @@ func UserFromRequest(st *state.State, req *http.Request) (*auth.UserState, error
 type metarepo interface {
 	Snap(string, string, store.Authenticator) (*snap.Info, error)
 	FindSnaps(string, string, store.Authenticator) ([]*snap.Info, error)
+	Updates([]string, store.Authenticator) ([]*snap.Info, error)
```

Comment:
when we switch to the new endpoint we should revisit whether it's updates or refreshes there as well. I think storeside it is still updates; the distinction snapside is that a refresh will (when everything is done) not hit the network, whereas update will. That is: updating is slow, hits the network, downloads stuff, but doesn't (necessarily) refresh. Refresh doesn't (necessarily) hit the network. Klaar?